pkg/kyverno/common/common.go

```diff
@@ -489,16 +489,18 @@ func ApplyPolicyOnResource(policy *v1.ClusterPolicy, resource *unstructured.Unst
 	resPath := fmt.Sprintf("%s/%s/%s", resource.GetNamespace(), resource.GetKind(), resource.GetName())
 	log.Log.V(3).Info("applying policy on resource", "policy", policy.Name, "resource", resPath)
 
+	resourceRaw, _ := resource.MarshalJSON()
```

Comment:
We should not skip the error check here. It might lead to undefined behavior if the Marshal failed.